### PR TITLE
Remove entirely infura secrets

### PR DESCRIPTION
**Pull request by dawidgil**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Example of .env file. Fill below secrets for local development.
+#INFURA_PROJECT_ID=
+#INFURA_PROJECT_SECRET=
```
Comment (jrhender) [inline .env.example:3]: Could you update the README with instructions of what one should do with `.env.example` in order to run Switchboard locally please?

File: src/app/shared/services/env/env.service.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export class EnvService {
   blockExplorerUrl: string = environment.blockExplorerUrl;
   SENTRY_ENVIRONMENT: string = environment.SENTRY_ENVIRONMENT;
   INFURA_PROJECT_ID: string =
-    INFURA_PROJECT_ID ?? environment?.INFURA_PROJECT_ID;
+    INFURA_PROJECT_ID ?? process?.env?.INFURA_PROJECT_ID;
   INFURA_PROJECT_SECRET: string =
-    INFURA_PROJECT_SECRET ?? environment?.INFURA_PROJECT_SECRET;
+    INFURA_PROJECT_SECRET ?? process?.env?.INFURA_PROJECT_SECRET;
```
Comment (jrhender) [inline src/app/shared/services/env/env.service.ts:26]: Do we also need to change the GHA jobs or will the secrets also be available from `process.env` (it seems like they were previously available via `environment`)?

**Review by jrhender — COMMENTED** [review]
Added a couple questions

File: src/app/shared/services/env/env.service.ts (hunk 2 above)
Comment (dawidgil) [reply src/app/shared/services/env/env.service.ts:26]: No need to change GHA jobs. This process.env is available locally and it is reading .env file. On GHA there is no file like this, so this part is not changed at all.

File: .env.example (hunk 1 above)
Comment (dawidgil) [reply .env.example:3]: Should i add link to the confluence page? Or just information that user needs to provide those values?

Comment (jrhender) [reply .env.example:3]: Hmm, good question. I think the instructions can assume that the person has Infura credentials. I think we need some steps like:

1. Execute `cp .env.example .env`
2. Fill in necessary variables

Does that make sense from your perspective?

Comment (dawidgil) [reply .env.example:3]: Yup, added to the readme. Hope the place where i added it is good in you opinion.

File: README.md [hunk 3]
```diff
@@ -31,6 +31,12 @@ Install dependencies:
 npm install
 ```
 
+### Running
+
+Before you start You need to provide some secrets:
+1. Execute cp .env.example .env
```
Comment (jrhender) [inline README.md]:
```suggestion
1. Execute `cp .env.example .env`
```

File: .env.example (hunk 1 above)
Comment (jrhender) [reply .env.example:3]: Yes, looks good, thanks!

**Review by jrhender — APPROVED** [review]
Minor suggestion to add backtick to instruction, but looks good, thanks!

**Pull request merged**